docs/source/getting-started.rst

```diff
@@ -0,0 +1,87 @@
+===============
+Getting Started
+===============
+
+This section provides an introduction on how to get started with using the SymbiFlow toolchain.
+In order to generate a bitstream (or any intermediate file format),
+you can use one of the toolchain tests. Following steps describe the whole
+process:
+
+Clone repository
+----------------
+
+.. code-block:: bash
+
+    git clone https://github.com/SymbiFlow/symbiflow-arch-defs.git
+
+Prepare environment
+-------------------
+
+Download all needed packages and databases content
+into the separated environment
+
+.. code-block:: bash
+
+    cd symbiflow-arch-defs
+    make env
+
+Build example
+-------------
+
+Enter the appropriate test build directory, depending on your target
+architecture and invoke the appropriate make target.
+
+Build directories depend on the architecture. Because of that,
+depending on the chosen target, a different toolchain backend will be used.
+(See `Project X-Ray <https://prjxray.readthedocs.io/en/latest/>`_
+and `Project Trellis <https://prjtrellis.readthedocs.io/en/latest/>`_
+for more information)
+
+Moreover, it is worth to note that target names have the form <*testname_platform_outputformat*>.
+
+Assuming that you would like to generate bitstream ``.bit`` file with
+the counter example for the arty board, which uses Xilinx Artix-7 FPGA,
+you will type:
+
+.. code-block:: bash
+
+    cd build/xc7/tests/counter
+    make counter_arty_bin
+
+Load bitstream
+--------------
+
+The last step is to load the bitstream to your platform.
+The final output file can be found in the appropriate test directory, i.e:
+``/build/xc7/tests/counter/counter_arty/artix7-xc7a50t-arty-roi-virt-xc7a50t-arty-test/top.bit``
+
+The loading proces may be different for every vendor.
+For convenience a Makefile target is provided for this purpose, e.g.:
```

Comment:
Just a "target", not "Makefile target".  Makefiles are only one of many CMake generators that can be used.

Reply:
Fixed